src/runtime/methodbinder.cs

```diff
@@ -335,6 +356,46 @@ internal Binding Bind(IntPtr inst, IntPtr args, IntPtr kw, MethodBase info, Meth
                     continue;
                 }
 
+                var matchedMethod = new MatchedMethod(kwargsMatched, defaultsNeeded, margs, outs, mi);
+                argMatchedMethods.Add(matchedMethod);
+            }
+            if (argMatchedMethods.Count() > 0)
```

Comment:
Please use `Count` property. `Count()` method might require a run through the entire collection.